src/test/javascript/spec/component/shared/clone-repo-button.component.spec.ts

```diff
@@ -155,6 +156,33 @@ describe('JhiCloneRepoButtonComponent', () => {
         expect(url).toEqual(info.versionControlUrl!);
     });
 
+    it('should insert the correct token in the repository url', () => {
+        component.user = { login: 'user1', guidedTourSettings: [], vcsAccessToken: 'token' };
+        component.repositoryUrl = `https://${component.user.login}@bitbucket.ase.in.tum.de/scm/ITCPLEASE1/itcplease1-exercise-team1.git`;
+        component.useSsh = false;
+        component.isTeamParticipation = false;
+
+        component.versionControlAccessTokenRequired = true;
+
+        // Placeholder is shown
+        let url = component.getHttpOrSshRepositoryUrl();
+        expect(url).toEqual(`https://${component.user.login}:**********@bitbucket.ase.in.tum.de/scm/ITCPLEASE1/itcplease1-exercise-team1.git`);
```

Comment:
```suggestion
        expect(url).toBe(`https://${component.user.login}:**********@bitbucket.ase.in.tum.de/scm/ITCPLEASE1/itcplease1-exercise-team1.git`);
```
The coding conventions mention that `toBe` should be used for assertions on simple datatypes.